lib/protocol/events.js

```diff
@@ -203,12 +203,13 @@ function eventsProtocolFactory (
     };
 
 
-    EventsProtocol.prototype.publishGraphStarted = function (graphid, status, nodeId) {
+    EventsProtocol.prototype.publishGraphStarted = function (graphid, data, nodeId) {
         var self = this;
         assert.uuid(graphid);
-        assert.string(status);
-
-        var data = { status: status };
+        assert.object(data);
+        assert.uuid(data.graphId);
```

Comment:
So the function `publishExternalEvent ` you commited before(https://github.com/RackHD/on-core/blob/master/lib/protocol/events.js#L312-L318)  should either add check function or don't check the data to let it more tolerance.
I think we should keep the format aligned here.